x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.test.tsx

```diff
@@ -124,33 +125,38 @@ describe('<CspPolicyTemplateForm />', () => {
     packageInfo?: PackageInfo;
     onChange?: jest.Mock<void, [NewPackagePolicy]>;
     agentlessPolicy?: AgentPolicy;
-  }) => (
-    <ThemeProvider theme={() => ({ eui: { euiSizeXS: '4px' } })}>
-      <TestProvider>
-        {edit && (
-          <CspPolicyTemplateForm
-            policy={newPolicy as PackagePolicy}
-            newPolicy={newPolicy}
-            onChange={onChange}
-            packageInfo={packageInfo}
-            isEditPage={true}
-            agentPolicy={agentPolicy}
-            agentlessPolicy={agentlessPolicy}
-          />
-        )}
-        {!edit && (
-          <CspPolicyTemplateForm
-            newPolicy={newPolicy}
-            onChange={onChange}
-            packageInfo={packageInfo}
-            isEditPage={false}
-            agentPolicy={agentPolicy}
-            agentlessPolicy={agentlessPolicy}
-          />
-        )}
-      </TestProvider>
-    </ThemeProvider>
-  );
+  }) => {
+    const { AppWrapper: FleetAppWrapper } = createFleetTestRendererMock();
+    return (
+      <ThemeProvider theme={() => ({ eui: { euiSizeXS: '4px' } })}>
+        <FleetAppWrapper>
```

Comment:
Just wondering if with `FleetAppWrapper` we could remove the `ThemeProvider` since we added it to support the lazy loaded fleet component `LazyPackagePolicyInputVarField`

Reply:
Tests passed locally with `ThemeProvider` removed as `FleetAppWrapper` contains its own `ThemeProvider`. See https://github.com/elastic/kibana/pull/178045/commits/ec0ed7942de28358f6a26c942aa78520b6f75401.

Reply:
Thanks!